rules/nette/src/Rector/ClassMethod/RemoveParentAndNameFromComponentConstructorRector.php

```diff
@@ -40,6 +39,14 @@ final class RemoveParentAndNameFromComponentConstructorRector extends AbstractRe
      */
     private const NAME = 'name';
 
+    /**
+     * Package "nette/application" is required for DEV, might not exist for PROD.
+     * So access the class throgh the string
+     *
+     * @var string
+     */
+    private const CONTROL_CLASS = '\Nette\Application\UI\Control';
```

Comment:
The first slash is not needed, as it's not valid class name. Other cases too

Reply:
Ok, fixing now